### PR TITLE
'basic link' button and inline with number in Coach

### LINKED ISSUE #8629: 'view learners' button in Coach  has incorrect styling
## Observed behavior

The 'view learners' button is styled as a raised secondary button:

image

## Expected behavior

Button should probable be a 'basic link' and inline with number, not on next line

## User-facing consequences

* button has more prominence on the page than appropriate
* too much space is taken up for functionality

## Steps to reproduce

view a class home page in Coach

## Context

0.15.0-beta1

### PR DESCRIPTION
**Pull request by abhimnc**
## Summary
Fixes #8629 

view learners' button in Coach has incorrect styling
Button is a 'basic link' and inline with number, not on next line
image

## References

https://github.com/learningequality/kolibri/issues/8629

## Reviewer guidance

is it okay? Do I need modify more?
image

----

## Testing checklist

- [-] Contributor has fully tested the PR manually
- [-] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [-] PR has the correct target branch and milestone
- [-] PR has 'needs review' or 'work-in-progress'   ##label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by rtibbles** [comment]
You have based your work here off the develop branch, which includes commits not in the release-v0.15.x branch.

To fix this, I'd recommend making sure your local copy of release-v0.15.x is up to date, and then do the following:
```
git checkout my-awesome-changes
git rebase -i release-v0.15.x
```
The second command here will open a text editor and prompt you with a list of commits. You should mark all the commits as 'drop' except your `added alignment space` commit.

Once that is done, you should force push to the remote `my-awesome-changes` branch, which will update this PR.

File: package.json [hunk 1]
```diff
@@ -63,5 +63,8 @@
   },
   "browserslist": [
     "extends browserslist-config-kolibri"
-  ]
+  ],
+  "dependencies": {
+    "caniuse-lite": "^1.0.30001280"
+  }
```
Comment (indirectlylit) [inline package.json]: Please revert the changes to package.json and yarn.lock as they are not directly related to this fix.

I've opened a follow-up issue related to the warnings you were seeing here: https://github.com/learningequality/kolibri/issues/8728

File: kolibri/plugins/coach/assets/src/views/home/HomePage/OverviewBlock.vue [hunk 2]
```diff
@@ -33,17 +33,13 @@
...
-            appearance="raised-button"
-            :to="classLearnersListRoute"
-          />
+          <template v-if="learnerNames.length > 0">
+            <KRouterLink
+              :text="$tr('viewLearners')"
+              appearance="basic-link"
+              :to="classLearnersListRoute"
+            />
+          </template>
```
Comment (indirectlylit) [inline kolibri/plugins/coach/assets/src/views/home/HomePage/OverviewBlock.vue]: please add a `margin-left: 24px` for balanced spacing:

| current | intended |
|--|--|
| image | image |

**Review by indirectlylit — CHANGES_REQUESTED** [review]
Thank you!

A couple small changes requested

File: kolibri/plugins/coach/assets/src/views/home/HomePage/OverviewBlock.vue (hunk 2 above)
Comment (abhimnc) [reply kolibri/plugins/coach/assets/src/views/home/HomePage/OverviewBlock.vue]: added in next commit.

File: package.json (hunk 1 above)
Comment (abhimnc) [reply package.json]: I am not able to revert changes to package.json and yarn.lock by git commands. should i delete this branch and create a new one with applied changes.

Comment (indirectlylit) [reply package.json]: Hi – I force-pushed an update to your branch which keeps only the relevant files.

For future reference, it's possible to break a previous commit in pieces and choose only portions of that commit using an interactive rebase.

See for example: https://stackoverflow.com/questions/6217156/break-a-previous-commit-into-multiple-commits

**Review by indirectlylit — APPROVED** [review]
looks good!

**Pull request merged**